src/sql/platform/tasks/browser/tasksRegistry.ts

```diff
@@ -58,8 +58,8 @@ export const TaskRegistry: ITaskRegistry = new class implements ITaskRegistry {
 			iconClass = this.taskIdToIconClassNameMap.get(item.id);
 		} else if (item.iconLocation) {
 			iconClass = ids.nextId();
-			createCSSRule(`.icon.${iconClass}`, `background-image: url("${(item.iconLocation.light || item.iconLocation.dark).toString()}")`);
-			createCSSRule(`.vs-dark .icon.${iconClass}, .hc-black .icon.${iconClass}`, `background-image: url("${(item.iconLocation.dark).toString()}")`);
+			createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(item.iconLocation.light || item.iconLocation.dark)}`);
```

Comment:
Would be useful to check if these itemLocations are properly setup as well. Sometimes they're not depending on the contribution point.

Reply:
Not sure what we can do if they aren't so I'm going to leave it as is right now - if they aren't there then we'll just not display anything which seems fine as a fallback.